packages/@coorpacademy-player-store/src/utils/state-extract.js

```diff
@@ -237,16 +237,30 @@ export const getPrevStepContent = (state: State): Content | void => {
   return progression.state.content;
 };
 
-export const getCurrentChapterId = (state: State): string | void => {
-  const slide = getCurrentSlide(state);
-
+export const getChapterIdBySlide = (state: State, slideId: string): string | void => {
+  const slide = getSlide(slideId)(state);
   if (!slide) {
     return;
   }
-
   return getChapterId(slide);
 };
 
+export const getCurrentChapterId = (state: State): string | void => {
+  const progression = getCurrentProgression(state);
+  if (!progression || !progression.state || !progression.state.nextContent) {
+    return;
+  }
+  if (progression.state.nextContent.type === 'slide') {
+    return getChapterIdBySlide(state, progression.state.nextContent.ref);
+  } else {
+    if (!progression.state.slides || isEmpty(progression.state.slides)) {
+      return;
+    }
+    const lastSlideId = last(progression.state.slides);
+    return getChapterIdBySlide(state, lastSlideId);
+  }
+};
```

Comment:
same for this

Reply:
=> https://github.com/CoorpAcademy/components/pull/1617/files#diff-66e631ed454627bf02a2067a526bf492R803-R810
C'est dans le cas où on est sur un extraLife, le nextContent n'est plus la slide.

Reply:
`popinEnd` uses the `getLives` function to determine whether to show/hide life

=> https://github.com/CoorpAcademy/components/blob/master/packages/%40coorpacademy-player-web/src/map-state-to-props/popin-end.js#L79

=> https://github.com/CoorpAcademy/components/blob/master/packages/%40coorpacademy-player-store/src/utils/state-extract.js#L460

Reply:
ok 👍